dlpar/dlpar_api/api.py

```diff
@@ -304,7 +485,9 @@ def linux_check_rm_cpu(self, linux_machine, quantity_before,
 
         # Also, verify if /proc/cpuinfo show all processors
         c_cmd = 'cat /proc/cpuinfo'
-        cmd = linux_machine.sshcnx.run_command(c_cmd).decode()
+        cmd = process.run(c_cmd).stdout()
+        self.log.debug(cmd)
+        # cmd = linux_machine.sshcnx.cmd(c_cmd).decode()
```

Comment:
any reason to keep this statement? If not can you remove it?

Reply:
This line is not needed, Removed in the latest commit.